### PR TITLE
finish index html, bug fix footer, all links work.

### PR DESCRIPTION
**Pull request by chuannan926**
Bug fix login link in nav-bar and FAQ button, product button, fix misspelling. 
Created a index-footer.html so that index.html works, 
all links work.

### CODE REVIEW

File: Rent-A-Rig/WebContent/html/index-footer.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+<link rel="stylesheet" href="../css/mainStyles.css">
```
Comment (Brain-Crumbs) [inline Rent-A-Rig/WebContent/html/index-footer.html]: the index-footer should link to css/mainStyles instead of ../css/mainStyles

**Review by Brain-Crumbs — CHANGES_REQUESTED** [review]
Everything looks great. Found one issue with a css link. but otherwise good. Fix this one issue and we can merge

**Review by Brain-Crumbs — APPROVED** [review]
looks good!

File: Rent-A-Rig/WebContent/html/index-footer.html (hunk 1 above)
Comment (chuannan926) [reply Rent-A-Rig/WebContent/html/index-footer.html]: Just change it ! Thanks!

**Pull request merged**